rest/tests/test_vocabulary.py

```diff
@@ -45,6 +45,22 @@ def test_vocabularies(self):
 
         self.create_vocabulary(self.repo.slug)
         vocabularies = self.get_vocabularies(self.repo.slug)
+
+        # assert that vocabularies are sorted.
+        try:
+            # for python 3
+            xrange
+        except NameError:
+            xrange = range
```

Comment:
Is this really necessary? Why not just use `range`? It's not going to be a long list in the test, is it?